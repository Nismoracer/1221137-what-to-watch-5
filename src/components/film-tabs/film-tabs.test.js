import React from "react";
import renderer from "react-test-renderer";
import FilmTabs from "./film-tabs";

const noop = () => {};

it(`Should render Film tabs correctly`, () => {
  const tree = renderer
    .create(
        <FilmTabs
          activeTab={`Overview`}
          onMenuClick={noop}
          renderActiveSection={noop}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
