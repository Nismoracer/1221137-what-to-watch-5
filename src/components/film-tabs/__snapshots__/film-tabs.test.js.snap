// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Film tabs correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
        id="Overview"
        onClick={[Function]}
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        id="Details"
        onClick={[Function]}
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        id="Reviews"
        onClick={[Function]}
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
